traffic_portal/app/src/common/modules/table/cacheGroupParameters/TableCacheGroupParamsUnassignedController.js

```diff
@@ -36,11 +36,13 @@ var TableProfileParamsUnassignedController = function(cg, parameters, $scope, $u
 	};
 
 	$scope.updateParams = function($event, paramId) {
-		var checkbox = $event.target;
-		if (checkbox.checked) {
+		var checkbox = $event.currentTarget.firstElementChild.firstElementChild; // finds the input element from the table row which was clicked
```

Comment:
this feels hacky. you are digging quite deep to get the checkbox

Reply:
i'm pretty sure there a click event on that checkbox you can simply catch or something

Reply:
can you do something like this instead? on tr click, force a click on the nested checkbox and then all the existing code can stay the same?